### PR TITLE
Update monitoring and logging guide

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates the monitoring and logging guide for Scalar products on a Kubernetes cluster.
Also, this PR adds sample custom values files of `kube-prometheus-stack` and `loki-stack` for Scalar product monitoring and logging.
Those sample files include some recommended/required configurations in the production environment.

Please take a look!

### CODE REVIEW

File: conf/scalar-loki-stack-custom-values.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    # scalar-labs.com/dedicated-node: scalardl-ledger
+    # scalar-labs.com/dedicated-node: scalardl-auditor
+
+  tolerations:
+    - effect: NoSchedule
+      key: scalar-labs.com/dedicated-node
+      operator: Equal
+      value: scalardb
+      # value: scalardl-ledger
+      # value: scalardl-auditor
```
Comment (kota2and3kan) [inline conf/scalar-loki-stack-custom-values.yaml:14]: In the production environment, it is recommended to set taints/tolerations to make specific worker nodes dedicated to Scalar products.
https://github.com/scalar-labs/scalar-kubernetes/blob/reorganize-docs/docs/CreateAKSClusterForScalarProducts.md#add-taint-to-the-worker-node-that-is-used-for-toleration

Since the promtail pods are deployed as DaemonSet, we need to add tolerations in the promtail configuration.
In other words, if we don't set tolerations in the custom values file of promtail, the promtail pods cannot be deployed on the worker node dedicated to Scalar products.

So, I added the `tolerations` setting in the sample custom values file.
With this tolerations configuration, the promtail pods can be deployed on the worker node dedicated to Scalar products.

**Comment by kota2and3kan** [comment]
I updated the Grafana data source configurations based on the suggestions in the following PR.
https://github.com/scalar-labs/helm-charts/pull/169

**Review by feeblefakie — COMMENTED** [review]
Left some minor suggestions, but LGTM other that those.
Thank you!

**Comment by kota2and3kan** [comment]
@feeblefakie  
Thank you for the suggestions!
I applied them. Please take a look again when you have time!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
